gensim/sklearn_integration/sklearn_wrapper_gensim_lsimodel.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+#
+"""
+Scikit learn interface for gensim for easy use of gensim with scikit-learn
+Follows scikit-learn API conventions
+"""
+import numpy as np
+
+from gensim import models
+from gensim import matutils
+from scipy import sparse
+from sklearn.base import TransformerMixin, BaseEstimator
+
+# accuracy defaults for the multi-pass stochastic algo
+P2_EXTRA_DIMS = 100  # set to `None` for dynamic P2_EXTRA_DIMS=k
+P2_EXTRA_ITERS = 2
+
+class SklearnWrapperLsiModel(models.LsiModel, TransformerMixin, BaseEstimator):
+    """
+    Base LSI module
+    """
+
+    def __init__(self, corpus=None, num_topics=200, id2word=None, chunksize=20000,
+                 decay=1.0, onepass=True, power_iters=P2_EXTRA_ITERS, extra_samples=P2_EXTRA_DIMS):
```

Comment:
please make defaults explicits. what is the reasons for using constant variables?